cssom-view-1/Overview.bs

```diff
@@ -1981,6 +1996,9 @@ Changes {#changes}
 This section documents some of the changes between publications of this specification. This section is not exhaustive. Bug fixes and editorial changes are
 generally not listed.
 
+<h3 id='changes-from-2022-07-07' class=no-num>Changes From 14 September 2023</h3>
```

Comment:
The ID here should match the title.

(It's also not clear to me why there are so many changes sections in this spec.  I'd normally expect one changes section for each new publication to https://www.w3.org/TR/cssom-view-1/ .)